tests/robustness/multiclient_test/framework/snapshotter.go

```diff
@@ -67,6 +74,20 @@ func (mcs *MultiClientSnapshotter) ConnectOrCreateRepo(repoPath string) error {
 	return err
 }
 
+// SetCacheSizeLimits sets the cache size limits for an existing repository
+// the repository server is connected to.
+// Allowed flags: content-cache-size-limit-mb, metadata-cache-size-limit-mb.
+func (mcs *MultiClientSnapshotter) SetCacheSizeLimits(cacheFlag string, cacheSize int) error {
+	if !strings.Contains(cacheFlag, contentCacheLimitMBFlag) && !strings.Contains(cacheFlag, metadataCacheLimitMBFlag) {
+		return ErrUnsupportedFlagsForCacheSet
+	}
```

Comment:
If we wanted to avoid the need for such validation, we could perhaps split it into two methods
- `SetContentCacheSizeLimit`
- `SetMetadataCacheSizeLimit`

that way we can easily see what we're trying to change without requiring the caller to pass in a particular string

Reply:
by doing so, we could also move
```
contentCacheLimitMBFlag  = "--content-cache-size-limit-mb"
metadataCacheLimitMBFlag = "--metadata-cache-size-limit-mb"
```

to this file, closer to where they're being used

Reply:
That seems like a good suggestion.
Also, the approach in this PR is different than the one used for the metadata repo. Can they be unified for consistency?

Reply:
@redgoat650  good point! I will implement the feedback.

@julio-lopez  I did think about using the same approach in the beginning. However, the repos are created in two different ways. The repo under test is created/acted upon using kopia cli and the metadata repo is created/acted upon by directly calling the go functions underneath. Unifying the approach to setting cache size would mean unifying the approach to rest of the interactions and updating any other code that depends on the current interface. That would be a much larger task and out of scope for this PR. 
What do you think?